src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />  
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Security.Principal.Windows.Tests</AssemblyName>
+    <RootNamespace>System.Security.Principal.Windows.Tests</RootNamespace>
+    <ProjectGuid>{6C36F3AC-54A1-4021-9F5D-CDEFF7347277}</ProjectGuid>
```

Comment:
Please add an <UnsupportedPlatforms/> attribute here for both Linux and OSX... these tests should be Windows only and will fail if run elsewhere.